src/Test.Utilities/AdditionalMetadataReferences.cs

```diff
@@ -35,20 +34,9 @@ public static class AdditionalMetadataReferences
         public static ReferenceAssemblies DefaultWithWinForms { get; } = Default
             .AddAssemblies(ImmutableArray.Create("System.Windows.Forms"));
 
-        public static MetadataReference SystemCollectionsImmutableReference { get; } = MetadataReference.CreateFromFile(typeof(ImmutableHashSet<>).Assembly.Location);
```

Comment:
Instead of simply fixing the missing dll I thought that would be better to use the new standard way of referencing the dlls. Let me know if you think I should revert this.